src/site/content/en/fast/chrome-ux-report-bigquery/index.md

```diff
@@ -8,16 +8,18 @@ description: |
   dataset to extract insightful results about the state of user experiences on
   the web.
 date: 2020-06-12
+updated: 2022-07-18
 tags:
   - performance
+  - chrome-ux-report
 ---
 
 The raw data of the Chrome UX Report
-([CrUX](https://developers.google.com/web/tools/chrome-user-experience-report/))
+([CrUX](https://developer.chrome.com/docs/crux/))
 is available on
 [BigQuery](https://console.cloud.google.com/bigquery?p=chrome-ux-report), a
 database on the Google Cloud Platform (GCP). Using BigQuery requires a
-[GCP project](https://developers.google.com/web/tools/chrome-user-experience-report/getting-started#getting-started)
+[GCP project](https://console.cloud.google.com/bigquery?p=chrome-ux-report&d=all&page=dataset)
```

Comment:
Not sure what else to put here in lieu of a more detailed CrUX-specific walkthrough

```suggestion
[GCP project](https://cloud.google.com/free)
```

Reply:
Yeah I struggled with this one myself. Maybe a mind in hindsight and we should expand this page with the detail from that link before it's gone? Happy to do that in this PR if you agree.

If not, then would prefer the link I suggested that the generic free one you suggested.

Reply:
I've opened a PR for the DCC docs to add this missing info: https://github.com/GoogleChrome/developer.chrome.com/pull/3250

Will update this PR to point to that.